src/cli.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const path = require('path')
+
+let inquirer = require('inquirer');
```

Comment:
Me agrada ver la definición de las variables así, const para cuando el valor no muta y let para cuando el valor muta
```suggestion
const fs = require('fs');
const path = require('path')

let inquirer = require('inquirer');
```